### DISCUSSION TITLE
Define the evaluation order of app-detection-rules

### DISCUSSION THREAD

**Discussion by mdicss** [Dynatrace/dynatrace-configuration-as-code#299] [Q&A] [answered]
It should be possible to define the order of app-detection-rules when defining them with monaco.  For now, there seems to be a random order in the resulting rules.

**Comment by tatomir146** [comment, marked as answer]
Hi @mdicss, there is already a ticket for this and it will be most probably solved by adding support for settings 2.0 api support. Please see #10
